descartes_light/test/src/perf_plotter.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+import math
+import numpy as np
+# import matplotlib as mpl
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import axes3d, Axes3D #<-- Note the capitalization! 
+import sys
+
+class Perf_Plotter():
```

Comment:
The convention for class names is `CamelCase`

```suggestion
class PerfPlotter():
```